src/apiCalls.js

```diff
@@ -0,0 +1,23 @@
+
+
+const getMovies = async(type, id) => {
+  let url = 'https://rancid-tomatillos.herokuapp.com/api/v2/movies'
+
+  if(id) {
+    url = `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`
+  } 
+  if (type === 'videos') {
+    url = `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/${type}`
+  }
+  
+  try {
+    const response = await fetch(url)
+    const data = await response.json() 
+    return data
+  } catch(error) {
+    console.log(error)
+    return error
```

Comment:
If we were to move the catch into the app.js file, we could do more with this error than console logging it - like setting it to state and displaying it to the user?